packages/meteor/server_environment.js

```diff
@@ -3,8 +3,11 @@ meteorEnv = {
   TEST_METADATA: process.env.TEST_METADATA || "{}"
 };
 
-if (typeof __meteor_runtime_config__ === "object") {
-  __meteor_runtime_config__.meteorEnv = meteorEnv;
+var config = typeof __meteor_runtime_config__ === "object" &&
```

Comment:
The `meteor` package is kinda primordial, so no `ecmascript`, but `const` would probably be fine here, since it's server code, and Meteor controls the Node version. Just need to make sure this version of the `meteor` package can't be installed in pre-Meteor-1.6 apps.

Reply:
The `meteor` package minor version was bumped to 1.9.0 in Meteor 1.7, so 1.9.x can't be installed in Meteor apps earlier than 1.7, which means this code will only ever run in Node 8 or later, which means `const` is safe here.